src/CoreWCF.NetNamedPipe/tests/CoreWCF.NetNamedPipe.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net6.0-windows;net7.0;net7.0-windows</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+    <TargetFrameworks>$(TestTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(TestTargetFrameworks);net6.0-windows;net7.0-windows;net8.0-windows</TargetFrameworks>
```

Comment:
You don't need this first TargetFrameworks property as it always gets overwritten by the second one now you removed the condition.

Reply:
good catch Thanks